tests/conftest.py

```diff
@@ -289,44 +274,6 @@ def geth_provider(networks):
         yield networks.provider
 
 
-@contextmanager
-def _isolation():
```

Comment:
deleting this and using Ape's fixture isolation injection system is the main reason for doing this